frontend/javascripts/oxalis/view/jobs/train_ai_model.tsx

```diff
@@ -407,42 +447,81 @@ function areInvalidAnnotationsIncluded<T extends HybridTracing | APIAnnotation>(
       "id" in annotation ? annotation.id : annotation.annotationId,
     );
     return {
-      areSomeAnnotationsInvalid: true,
-      invalidAnnotationsReason: `All annotations must have at least one bounding box. Annotations without bounding boxes are: ${annotationIds.join(", ")}`,
+      hasAnnotationErrors: true,
+      errors: [
+        `All annotations must have at least one bounding box. Annotations without bounding boxes are: ${annotationIds.join(", ")}`,
```

Comment:
Maybe also include newline hear for readability? 

```suggestion
        `All annotations must have at least one bounding box. Annotations without bounding boxes are:\n${annotationIds.join("\n")}`,
```
I would at least make a new line after the `:`. The newlines between annotationIds is just a maybe. I don't know how long such a list can become :)